sci2zero/TeslaRIS-frontend
# Feature/serial events

## Description

Added CRUD operations for event relations. Serial event logic is now sound and it has a different landing page layout.

## How should this be tested?

Login as either researcher or admin. Navigate to any event's page. Serial events should contain basic details, contributors and list of instance events. Regular events should have proceedings list, publications list and relations list which is editable. 

## Checklist
